src/utils/rewritePaths.js

```diff
@@ -24,7 +24,7 @@ export default function rewritePaths(error, file, contents, moduleContext, callb
 
   const rewritten = contents.replace(useRegexp, (entire, importer, single, double, unquoted) => {
```

Comment:
Not sure what `unquoted` is, or if we should support this as well?

Reply:
No idea.